app/admin/company.rb

```diff
@@ -21,6 +21,36 @@
         column :readable_locality
       end
     end
-  end
 
+    panel I18n.t('active_admin.territories.contacted_experts') do
```

Comment:
DRY this and the equivalent in the Facility page

Reply:
Let’s not: we’ll have proper UI for this, outside of ActiveAdmin hopefully soon enough, and not waste too much time configuring views in AA.